2-mandatory/2-retrying-random-numbers.js

```diff
@@ -10,7 +10,11 @@ function generateRandomNumber() {
 }
 
 function getRandomNumberGreaterThan50() {
-    // TODO - implement using a do-while loop
+    let random = generateRandomNumber();
```

Comment:
The exercise asks for a `do { } while` loop here :)

Reply:
Yes, using a do-while loop here will avoid this small bit of code duplication. Otherwise, it's looking good :)